# OpenFOAM-Extend needs a libMGridGen library, with an extra lib aratio.o (compared to linmgrid)
# May 18th 2020 by Balazs Hajgato (Ghent University)
diff -ru ParMGridGen-1.0-orig/MGridGen/Lib/Makefile ParMGridGen-1.0/MGridGen/Lib/Makefile
--- ParMGridGen-1.0-orig/MGridGen/Lib/Makefile	2001-12-05 00:01:00.000000000 +0100
+++ ParMGridGen-1.0/MGridGen/Lib/Makefile	2020-05-20 11:10:04.781008905 +0200
@@ -28,6 +28,8 @@
 IMOBJS = $(addprefix ../IMlib/, $(IOBJS)) 
 
 
+all: ../../libmgrid.a ../../libMGridGen.a
+
 .c.o:
 	$(CC) $(CFLAGS) -c $*.c
 
@@ -35,6 +37,10 @@
 	$(AR) $@  $(OBJS) $(IMOBJS)
 	$(RANLIB) $@
 
+../../libMGridGen.a: aratio.o $(OBJS) $(IMOBJS)
+	$(AR) $@  aratio.o $(OBJS) $(IMOBJS)
+	$(RANLIB) $@
+
 clean:
 	rm -f *.o
 
